api/plugin_helpers.go

```diff
@@ -33,56 +43,6 @@ const (
 	PluginUnwrapTokenEnv = "VAULT_UNWRAP_TOKEN"
 )
 
-// sudoPaths is a map containing the paths that require a token's policy
```

Comment:
Note to reviewer: All this removed code is moved without change to `sudo_paths.go`